### ISSUE TITLE
Travis failing -- not using Java8

### ISSUE THREAD

**Issue by julie-sullivan** [intermine/intermine_configurator#36]
I am getting this error:

```
> Task :compileJava

/home/travis/build/intermine/intermine_configurator/src/main/java/io/swagger/api/ApiResponseMessage.java:3: error: package javax.xml.bind.annotation does not exist

import javax.xml.bind.annotation.XmlTransient;

                                ^

/home/travis/build/intermine/intermine_configurator/src/main/java/io/swagger/api/ApiResponseMessage.java:6: error: package javax.xml.bind.annotation does not exist

@javax.xml.bind.annotation.XmlRootElement

                          ^

/home/travis/build/intermine/intermine_configurator/src/main/java/io/swagger/api/ApiResponseMessage.java:45: error: cannot find symbol

    @XmlTransient

     ^

  symbol:   class XmlTransient

  location: class ApiResponseMessage

Note: /home/travis/build/intermine/intermine_configurator/src/main/java/io/swagger/RFC3339DateFormat.java uses or overrides a deprecated API.

Note: Recompile with -Xlint:deprecation for details.

Note: Some input files use unchecked or unsafe operations.

Note: Recompile with -Xlint:unchecked for details.

3 errors
```

Only on travis, not locally.

https://stackoverflow.com/questions/52502189/java-11-package-javax-xml-bind-does-not-exist

It seems like this library was removed in Java 11. I specified Java 8 in the travis config files but doesn't seem to have worked?

**Comment by julie-sullivan** [comment]
i guess the real answer is to upgrade to Java 11 ( ... but i don't want to ...)

**Comment by yochannah** [comment]
fixed by #40
